generator/src/test/java/com/scottlogic/deg/generator/generation/FieldSpecValueGeneratorTests.java

```diff
@@ -263,10 +269,6 @@ void generate_fieldSpecContainsStringValuesForMustContainRestrictionAndFieldSpec
 
         Set<DataBag> expectedDataBags = new HashSet<>(
             Arrays.asList(
-                DataBag.startBuilding().set(
```

Comment:
Why is this no longer expected? The setup for the test doesn't appear to have changed.

Reply:
We thought that the test expected outcomes were previously incorrect. However, we are not entirely sure whether the inputs to this test make sense?